### PR TITLE
Removing audio stuff in favor of flame_audio

### PR DESCRIPTION
**Pull request by luanpotter**
# Description

Removing audio stuff in favor of flame_audio

Fixes: v1 milestone ticket

I haven't changed the docs yet because not sure where each part will live (flame_audio, flame, etc). Once we have a better idea of how the website will integrate with these artifacts I can refactor all the audio docs to this new structure.

## Type of change

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -6,10 +6,7 @@ homepage: https://github.com/flame-engine/flame
 dependencies:
   flutter:
     sdk: flutter
-  audioplayers: ^0.15.1
   ordered_set: ^2.0.0
-  path_provider: ^1.6.0
```
Comment (luanpotter) [inline pubspec.yaml:11]: wasn't even used! \o/

**Review by spydon — APPROVED** [review]
Nice, lgtm

File: lib/flame.dart [hunk 2]
```diff
@@ -1,40 +1,30 @@
...
   static AssetBundle _bundle;
   static AssetBundle get bundle => _bundle == null ? rootBundle : _bundle;
 
-  /// Access a shared instance of the [FlameAudio] class.
-  static FlameAudio audio = FlameAudio();
-
-  /// Access a shared instance of the [Bgm] class.
-  static Bgm _bgm;
-  static Bgm get bgm => _bgm ??= Bgm();
+  /// Access a shard instance of [AssetsCache] class.
```
Comment (erickzanardo) [inline lib/flame.dart:19]: I know this was not written by you, but there is a typo here: `shared`

File: pubspec.yaml (hunk 1 above)
Comment (erickzanardo) [reply pubspec.yaml:11]: \o/

**Review by erickzanardo — APPROVED** [review]
One nit, but LGTM already.

I think we could keep our doc here, and I would just point out that there is an underlying package that should be used to enable the audio features.

File: lib/flame.dart (hunk 2 above)
Comment (luanpotter) [reply lib/flame.dart]: boy scout's rule!

**Pull request merged**